sdks/python/apache_beam/typehints/typehints.py

```diff
@@ -296,19 +303,21 @@ def __getitem___(self, py_type):
 
 
 def validate_composite_type_param(type_param, error_msg_prefix):
-  """Determines if an object is a valid type parameter to a CompositeTypeHint.
+  """Determines if an object is a valid type parameter to a
+  :class:`CompositeTypeHint`.
+
+  Implements sanity checking to disallow things like::
 
-  Implements sanity checking to disallow things like:
-    * List[1, 2, 3] or Dict[5].
+    List[1, 2, 3] or Dict[5].
 
   Args:
     type_param: An object instance.
-    error_msg_prefix: A string prefix used to format an error message in the
-      case of an exception.
+    error_msg_prefix (:class:`str`): A string prefix used to format an error
+      message in the case of an exception.
 
   Raises:
-    TypeError: If the passed 'type_param' is not a valid type parameter for a
-      CompositeTypeHint.
+    :class:`~exceptions.TypeError`: If the passed **type_param** is not a valid
```

Comment:
Without it, even though it is parsed correct, it generates a broken link because it is looking for ``:exc:`TypeError` `` instead of ``:class:`TypeError` ``. However, after further testing I can get rid of the ``:class:`` prefix and just use ``~exceptions.TypeError`` and that will work. I'll change all instances of this.